### DISCUSSION TITLE
Exact match on custom column

### DISCUSSION THREAD

**Discussion by tonycc0606** [microsoft-search/pnp-modern-search#2336] [Q&A]
Hello, I have created some custom columns on the document library and create managed properties for those columns. Complete match is not selected for these managed properties, however, when performing search, Search Results web part only returns results if it's exact match (text typed in the search box web part). For example, if one of the document has AAA under TestColumnA, typing AA in the Search Box web part will not return anything, only AAA can return results... Is this expected behaviour?

**Comment by wobba** [comment]
This is expected. You need to append `*` to trigger wildcard match. See https://docs.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference#using-words-in-the-free-text-kql-query
